engines/ospray/ispc/render/utils/SkyBox.ispc

```diff
@@ -66,9 +66,9 @@ vec4f skyboxMapping(const uniform Renderer* uniform renderer,
 
     // Only st needs to be set to sample from Texture2D
     varying DifferentialGeometry dg;
-    dg.st = make_vec2f(
-        ((atan2(intersection.x, intersection.z) / PI) + 1.f) * .5f,
-        -(asin(intersection.y) / PI) + .5f);
+    dg.st =
+        make_vec2f(((atan2(-intersection.x, intersection.z) / PI) + 1.f) * .5f,
+                   -(asin(intersection.y) / PI) + .5f);
```

Comment:
I will trust you on this, but shouldn't there be a small test?

Reply:
Will have a follow-up commit for this. Testing the environment map in general as well.